data/insecure_full.json

```diff
@@ -3897,6 +3897,15 @@
                 "<0.7.2"
             ],
             "v": "<0.7.2"
+        },
+        {
+            "advisory": "Cross-site scripting (XSS) vulnerability in the _keyify function in mistune.py in Mistune before 0.8.1 allows remote attackers to inject arbitrary web script or HTML by leveraging failure to escape the 'key' argument.",
+            "cve": 'CVE-2017-16876',
```

Comment:
@rohit-jamuar I assume it's failing because of `'`, it's not valid format in JSON